pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/adhoc/GjenlevendeInfoEtter1970.kt

```diff
@@ -73,38 +73,38 @@ object GjenlevendeInfoEtter1970 : AutobrevTemplate<Unit> {
                 text(
                     Bokmal to "Trenger du hjelp til å få ny jobb eller jobbe mer?",
                     Nynorsk to "Treng du hjelp til å få ny jobb eller jobb meir?",
-                    English to "Do you need help to get employment?",
+                    English to "Do you need help getting a new job or more work?",
                 )
             }
 
             paragraph {
                 text(
                     Bokmal to "NAV tilbyr ulike tjenester og støtteordninger for deg som trenger hjelp til å få jobb. Du kan finne jobbsøkertips og lenke til ledige jobber på $ARBEID_URL.",
                     Nynorsk to "NAV tilbyr ulike tenester og støtteordningar for deg som treng hjelp til å få jobb. Du kan finne jobbsøkjartips og lenkje til ledige jobbar på $ARBEID_URL.",
-                    English to "The Norwegian Labour and Welfare Administration (NAV) offers various services and support schemes for those who need help to find employment. You can find jobseeker tips and links to available jobs at $ARBEID_URL.",
+                    English to "The Norwegian Labour and Welfare Administration (NAV) offers various services and support schemes for those who need help finding a job. You can find job search tips and a hyperlink to open jobs online at $ARBEID_URL.",
```

Comment:
"You can find job search tips and a hyperlink to open jobs online at $ARBEID_URL." Did you copy from the old design? this seems like a regression.